src/patterns/gen-ai/aws-model-deployment-sagemaker/jumpstart-sagemaker-endpoint.ts

```diff
@@ -28,6 +28,7 @@ export interface JumpStartSageMakerEndpointProps {
   readonly vpcConfig?: sagemaker.CfnModel.VpcConfigProperty | undefined;
   readonly environment?: { [key: string]: string };
   readonly startupHealthCheckTimeoutInSeconds?: number;
+  readonly acceptEula?: boolean;
```

Comment:
Is the `acceptEula` optional at this point?

Reply:
I am waiting to get some feedback on which models are concerned, in the meantime I'll put back this PR in draft mode

Reply:
As discussed, keeping it optional and added a check to verify if it is required for each model in latest commit